my-site.rb

```diff
@@ -1,22 +1,38 @@
 require "sinatra"
 require "sinatra/reloader"
+require './lib/marleigh'
 
 class MySite < Sinatra::Base
   register Sinatra::Reloader
 
+
   get "/" do
-    send_file'public/Home.html'
+    @marleigh = Marleigh::Page.me
+    erb :index
+  end
+
+  get "/:name" do
+    @page = Marleigh::Page.new(params[:name])
+  end
```

Comment:
I think this route here might be what's causing you problems. the `:name` part will match anything - `/kerri` or `/bubblegum` or whatever. When a new page is requested, Sinatra goes from the top of this file to the bottom, matching the URL to each route defined here, until it matches one. Soooooo... `/about_me` gets clicked in the browser, it doesn't match `/` on line 9, but it does match `/:name` here, and so Sinatra executes that code, even though you have an exact match for `/about_me` further on.... its not that smart :)

Try putting lines 14-16 at the end of the list and see what happens; I think it might work!